src/main/java/svc/managers/SitemapManager.java

```diff
@@ -0,0 +1,36 @@
+package svc.managers;
+
+import org.springframework.stereotype.Component;
+import svc.models.Sitemap;
+import svc.models.SitemapUrl;
+
+@Component
+public class SitemapManager {
+	
+	public Sitemap generateSitemap() {
+		Sitemap sitemap = new Sitemap();
```

Comment:
Also above is using the dynamic path of the app instead of hard-coding production. Plus, it leaves the app URL from duplication across the app.